apps/src/templates/manageStudents/DownloadParentLetters.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Button from '@cdo/apps/templates/Button';
+import ReactTooltip from 'react-tooltip';
+import i18n from '@cdo/locale';
+import {teacherDashboardUrl} from '@cdo/apps/templates/teacherDashboard/urlHelpers';
+
+const styles = {
+  button: {
+    marginLeft: 'auto'
+  }
+};
+
+export default class DownloadParentLetters extends Component {
+  static propTypes = {
+    numStudents: PropTypes.number.isRequired,
+    sectionId: PropTypes.number
+  };
+
+  render() {
+    return (
+      <div style={styles.button}>
+        <span data-tip="" data-for="download-letter">
+          <Button
+            __useDeprecatedTag
+            href={teacherDashboardUrl(this.props.sectionId, '/parent_letter')}
+            target="_blank"
+            color={Button.ButtonColor.gray}
+            text={i18n.downloadParentLetter()}
+            icon="file-text"
+          />
+        </span>
+        <ReactTooltip
+          id="download-letter"
+          role="tooltip"
```

Comment:
:open_mouth: I couldn't find the `role` prop documented for `react-tooltip` and was about to complain, but then ran across [this pull request](https://github.com/wwayne/react-tooltip/pull/172/files) on their repo.  You should totally open a PR to update their README and document that role and aria props are supported!